content/en/about/contributing/shortcodes/_index.md

```diff
@@ -67,26 +66,42 @@ an optional highlighting type parameter.
 The supported languages are dependant on the syntax highlighter, which depends
 itself on the mardkown parser.
 
-{{% alert title="Note" color="info" %}} For [Hugo](https://gohugo.io/) the
-[goldmark parser](https://github.com/yuin/goldmark) is used and it relies on the
+{{% alert title="Note" color="info" %}} [Hugo](https://gohugo.io/) uses the
+[goldmark parser](https://github.com/yuin/goldmark), which relies on
 [Prism syntax highlighting](https://prismjs.com/download.html#themes=prism).
 {{% /alert %}}
 
-The following code should be used for a shell excerpt:
+The following markdown creates a shell excerpt:
 
+<!-- markdownlint-disable commands-show-output -->
 ````markdown
 ```shell
-ssh-keygen -f fedcloud
-echo $HOME
+$ ssh-keygen -f fedcloud
+$ echo $HOME
 ```
 ````
 
 Will render as:
 
 ```shell
-ssh-keygen -f fedcloud
-echo $HOME
+$ ssh-keygen -f fedcloud
+$ echo $HOME
 ```
+<!-- markdownlint-enable commands-show-output -->
+
+{{% alert title="Tip" color="info" %}} If you click the _Copy_ button in the
+top-right corner of a shell example, all commands in that block are copied to
+the clipboard. The prompt in front of each command, and any command
+output is not copied.
+{{% /alert %}}
+
+{{% alert title="Note" color="info" %}} In case the command(s) in your
+shell example cause the introduction of a horizontal scroll bar, [consider
```

Comment:
I would be stricter here, saying that people should try to have commands that are not longer than 80 chars.
Seeing scrolls bar means you need to see the rendered output and also it will depend on your window size.

Reply:
This is already stated in the basic rules (where I just added the suggested char limit of 80)